### PR TITLE
Update playlist.xml

### PR DESCRIPTION
**Pull request by vanichitkara**
Divided hotspot_all into 4 targets, hotspot_compiler, hotspot_gc, hotspot_runtime and hotspot_serviceability targets. Resolves issue #2477

### CODE REVIEW

**Review by sophia-guo — APPROVED** [review]
Grinder: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/

LGTM

**Comment by smlambert** [comment]
Link from comment above: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/13/

Another link to test against openj9 hotspot_serviceability: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/15

File: openjdk/playlist.xml [hunk 1]
```diff
@@ -64,6 +120,31 @@
...
+			<variation>Mode150</variation>
+			<variation>Mode650</variation>
+			<variation>Mode1000</variation>
+		</variations>
+		<command>$(JAVA_COMMAND) -Xmx512m -jar $(Q)$(TEST_RESROOT)$(D)jtreg$(D)lib$(D)jtreg.jar$(Q) \
+	$(JTREG_BASIC_OPTIONS) $(JVM_NATIVE_OPTIONS) -vmoptions:$(Q)-Xmx512m $(JVM_OPTIONS)$(Q) \
+	-w $(Q)$(REPORTDIR)$(D)work$(Q) \
+	-r $(Q)$(TEST_RESROOT)$(D)report$(Q) \
+	-jdk:$(Q)$(TEST_JDK_HOME)$(Q) \
+	$(Q)$(JTREG_HOTSPOT_TEST_DIR): hotspot_serviceability$(Q); \
```
Comment (smlambert) [inline openjdk/playlist.xml:135]: need to remove the whitespace between ':' and 'hotspot_serviceability' (I see that I had actually added it in the example in the issue, but it won't work with the whitespace present)

**Review by smlambert — COMMENTED** [review]
One minor change required, otherwise LGTM

File: openjdk/playlist.xml (hunk 1 above)
Comment (vanichitkara) [reply openjdk/playlist.xml]: I removed the whitespace, I hope it will work now.

**Comment by smlambert** [comment]
Grinder for hotspot_serviceability against openj9:
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/53/console

```
15:54:16  Error: Group spec does not specify a valid test suite: /home/jenkins/workspace/Grinder/openjdk-tests/TKG/../openjdk/openjdk-jdk/hotspot/test:hotspot_serviceability
15:54:16  
15:54:16  hotspot_serviceability_1_FAILED
```

I see that the hotspot folder is removed from https://github.com/ibmruntimes/openj9-openjdk-jdk8 so enabling these tests for openj9 as I thought is not so simple.  (They can be run by setting JDK_REPO to point to a non-extensions repo, but that will not work as a general approach).  


For this reason, @vanichitkara please add the 
```
		<impls>
			<impl>hotspot</impl>
		</impls>
```
block to the hotspot_serviceability target as you have done with the other 3.  

Then we can merge this PR.

I will talk to openj9 team about options for them to run these tests against openj9 impl, but we will not worry about that for now.

**Comment by vanichitkara** [comment]
I've added the block to the hotspot_serviceability target. Please review it.

**Review by smlambert — APPROVED** [review]
thanks @vanichitkara !

**Pull request merged**